### PR TITLE
AUTH-119: SAML idp tests

### PR DESCRIPTION
**Pull request by kconkas**
This is to merge AUTH-180 + several minor typos to master.

### CODE REVIEW

File: test/saml/pom.xml [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+		<plugins>
```
Comment (ehardon) [inline test/saml/pom.xml]: Maybe a minor code indentation here

File: test/saml/src/test/java/org/alfresco/identity/service/saml/test/TokenTest.java [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+        driver.get("https://" + getHostname() + "/auth/realms/" + getRealm() +"/protocol/openid-connect/auth?response_type=id_token%20token&client_id=alfresco&state=CIteJYtFrA22JnCikKHJ2QPrNuGHzyOphE1SsSNs&redirect_uri=http%3A%2F%2Flocalhost%2Fdummy_redirect&scope=openid%20profile%20email&nonce=CIteJYtFrA22JnCikKHJ2QPrNuGHzyOphE1SsSNs");
+       
+        //Click on SAML link on login page
+        WebElement element = driver.findElement(ByLinkText.linkText(TokenTestConstants.ELEMENT_SAML));
+        element.click();
+
+        //Select User, Enter password, and submit form on SAML page
+        Select select = new Select(driver.findElement(ByName.name(TokenTestConstants.ELEMENT_USERID)));
+        select.selectByVisibleText(getUser());
+        WebElement passwordfield = driver.findElement(ByName.name(TokenTestConstants.ELEMENT_PASWORD));
```
Comment (ehardon) [inline test/saml/src/test/java/org/alfresco/identity/service/saml/test/TokenTest.java]: should we modify the  `passwordfield` to corespond to the camel case standard?

Comment (kconkas) [reply test/saml/src/test/java/org/alfresco/identity/service/saml/test/TokenTest.java]: Thanks for spotting this, I've fixed it in 6159d3f.

File: test/saml/pom.xml (hunk 1 above)
Comment (kconkas) [reply test/saml/pom.xml]: Fixed indentation in b3f6ea2.

**Pull request merged**